code/modules/vore/eating/contaminate_vr.dm

```diff
@@ -18,7 +18,7 @@ var/image/gurgled_overlay = image('icons/effects/sludgeoverlay_vr.dmi')
 		name = "[gurgleflavor] [cleanname]"
 		desc = "[cleandesc] It seems to be covered in ominously foul residue and needs a wash."
 		for(var/obj/item/O in contents)
-			gurgle_contaminate(item_storage)
+			O.gurgle_contaminate(item_storage)
```

Comment:
Sorry, did everything but this fix in this PR my way instead, can you fix the merge conflict and I can merge this part?